hack/testing-olm/test-200-verify-es-metrics-access.sh

```diff
@@ -99,7 +99,7 @@ es_pod=$(oc -n ${TEST_NAMESPACE} get pod -l component=elasticsearch -o jsonpath=
 log::info "---------------------------------------------------------------"
 log::info "Waiting until elasticsearch cluster initialization is completed"
 log::info "---------------------------------------------------------------"
-try_until_success "oc -n ${TEST_NAMESPACE} exec ${es_pod} -c elasticsearch -- test -f /opt/app-root/src/init_complete" $((2 * $minute))
+expect_success "oc -n ${TEST_NAMESPACE} wait --for=condition=Ready pod/${es_pod} --timeout=120s"
```

Comment:
this is required to fix https://github.com/openshift/origin-aggregated-logging/pull/1893
its functionally the same but we will no longer have that file on the container